### PR TITLE
Added French translation

### PR DESCRIPTION
**Pull request by makruger**
@alarcher might want to verify this is correct.

### CODE REVIEW

**Comment by pyhalov** [comment]
@makruger: you'll likely want to also fix https://github.com/OpenIndiana/oi-userland/blob/oi/hipster/components/openindiana/openindiana-welcome/os-welcome.p5m

**Comment by alarcher** [comment]
Just nitpicking, the french typography has a space before '?', '!' and ':' unlike the english one: it is called 'espace insécable' and can be coded in html as #8239;
Otherwise looking good, thanks!

**Comment by makruger** [comment]
@alarcher Thanks for pointing that out. I was wondering about that.....and now I know. Also, I noticed << used where I would expect to see double quotes. Was that an anomaly or expected?

**Comment by makruger** [comment]
Fixed French translation and added Greek and Russian translations too.

**Comment by alarcher** [comment]
These << >> are called 'guillemets' they are the equivalent to double quotes in french typography.
Using double quotes as quotation marks is specific to the english typography, other languages may have different rules. https://en.wikipedia.org/wiki/Quotation_mark
It seems that you may put the fine space for padding between quotation marks, I did not know this.

File: data/html/ru/index.html [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+</head>
+<body>
+<div id="content">
+<h1><img src="../images/oi-logo.png" width=300 height=80
+alt="OpenIndiana Logo"></h1>
+
+<h2>Добро пожаловать в OpenIndiana Hipster!</h2>
+
+<p>ОС OpenIndiana - дистрибутив illumos, Unix-подобная операционная система, являющаяся прямым наследником ОС OpenSolaris.
+OpenIndiana Hipster создается силами сообщества и для сообщества как ОС общего назначения, обладающая набором уникальных возможностей.</p>
```
Comment (pyhalov) [inline data/html/ru/index.html]: OpenIndiana Hipster создаётся ...

File: data/html/ru/index.html [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+
+<a href="https://wiki.openindiana.org/oi/Release+Notes">https://wiki.openindiana.org/oi/Release+Notes</a>
+
+<br>
+<br>
+
+<h2>О нас</h2>
+
+<p>Проект OpenIndiana представляет собой сообщество добровольцев и людей, увлеченных Unix-системами, находящихся в разных частях земного шара.
+В настоящее время сервера для сборки проекта и хостинг предоставляется английской компанией EveryCity.</p>
```
Comment (pyhalov) [inline data/html/ru/index.html]: В настоящее время серверы для сборки проекта и хостинг предоставляется компанией EveryCity

**Comment by pyhalov** [comment]
Sorry, missed some stylistic issues...

**Comment by pyhalov** [comment]
Closing,  superseded by #4

**Pull request closed without merging**